WordPress/src/main/java/org/wordpress/android/ui/reader/services/update/ReaderUpdateJobService.java

```diff
@@ -0,0 +1,62 @@
+package org.wordpress.android.ui.reader.services.update;
+
+import android.annotation.TargetApi;
+import android.app.job.JobParameters;
+import android.app.job.JobService;
+
+import org.wordpress.android.WordPress;
+import org.wordpress.android.util.AppLog;
+
+import java.util.EnumSet;
+
+import static org.wordpress.android.ui.reader.services.update.ReaderUpdateServiceStarter.ARG_UPDATE_TASKS;
+
+@TargetApi(21)
+public class ReaderUpdateJobService extends JobService implements ServiceCompletionListener {
+    /***
```

Comment:
We have this comment here, in `ReaderUpdateService`, and `ReaderUpdateLogic`. I think we can remove it from both services, and re-word it in `ReaderUpdateLogic` to better reflect the purpose of that class.

Reply:
addressed in 20793d9